lib/percy/client/connection.rb

```diff
@@ -102,10 +110,43 @@ def post(path, data, options = {})
             sleep(rand(1..3))
             retry
           end
+
           raise e
         end
+
         JSON.parse(response.body)
       end
+
+      def _headers
+        {
+          'Content-Type' => 'application/vnd.api+json',
+          'User-Agent' => _user_agent,
+        }
+      end
+
+      def _user_agent
```

Comment:
Probably worth memoizing this so it doesn't have to recompute for every request.

Easy way:
```ruby
def ...
  return @_user_agent unless @_user_agent.nil?
  ...
  @_user_agent = "#{client} (#{environment})"
end